### ISSUE TITLE
Rounded rectangle tool

### ISSUE THREAD

**Issue by dsizzle** [HaikuArchives/ArtPaint#341]
The rectangle tool should have a "corner radius" option to allow rounded corners.

**Comment by humdingerb** [comment]
Yes. Possibly as a checkmark "Corner radius" and a BSpinner control und the existing "Fill rectangle".
